<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesView</class>
 <widget class="QWidget" name="PreferencesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#ColorPicker {
border-width: 1;
border-color: black;
background: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="labelLineNumbers">
     <property name="styleSheet">
      <string notr="true">QLabel {
font-weight: 600;
}</string>
     </property>
     <property name="text">
      <string>Line numbers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxLineNumbers">
     <property name="text">
      <string>Display line numbers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCurrentLine">
     <property name="styleSheet">
      <string notr="true">QLabel {
font-weight: 600;
}</string>
     </property>
     <property name="text">
      <string>Current Line</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxCurrentLine">
     <property name="text">
      <string>Highlight current line</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelBracket">
     <property name="styleSheet">
      <string notr="true">QLabel {
font-weight: 600;
}</string>
     </property>
     <property name="text">
      <string>Bracket Matching</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxBracket">
     <property name="text">
      <string>Highlight matching bracket</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
